src/main/java/org/usfirst/frc4904/auton/PickupAndShootSide.java

```diff
@@ -0,0 +1,32 @@
+package org.usfirst.frc4904.auton;
+
+import java.util.Arrays;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.FlywheelSpinDown;
+import org.usfirst.frc4904.robot.commands.RunIntake;
+import org.usfirst.frc4904.robot.commands.Shoot;
+import org.usfirst.frc4904.robot.commands.vision.VisionMoveHighPort;
+import org.usfirst.frc4904.standard.commands.chassis.SimpleSplines;
+
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+
+class PickupAndShootSide extends AutonRoutine {
+
+        public PickupAndShootSide() {
```

Comment:
instead of using `andThen`, you can use the `addCommands` method and pass in all your commands in order